apps/platforms/win/windows_terminal/windows_terminal.py

```diff
@@ -84,5 +84,69 @@ def file_manager_open_volume(volume: str):
     def tab_jump(number: int):
         actions.key("ctrl-alt-{}".format(number))
 
+    # user.splits implementation:
+
+    def split_window_right():
+        """Move active tab to right split"""
+        # TODO: decide whether this notification is good style
+        actions.app.notify("\"Split right\" is not possible in windows terminal without special configuration. Use \"split vertically\" instead.")
+
+    def split_window_left():
+        """Move active tab to left split"""
+        # TODO: decide whether this notification is good style
+        actions.app.notify("\"Split left\" is not possible in windows terminal without special configuration. Use \"split vertically\" instead.")
+
+    def split_window_down():
+        """Move active tab to lower split"""
+        # TODO: decide whether this notification is good style
+        actions.app.notify("\"Split down\" is not possible in windows terminal without special configuration. Use \"split horizontally\" instead.")
+
+    def split_window_up():
+        """Move active tab to upper split"""
+        # TODO: decide whether this notification is good style
+        actions.app.notify("\"Split up\" is not possible in windows terminal without special configuration. Use \"split horizontally\" instead.")
+
+    def split_window_vertically():
+        """Splits window vertically"""
+        actions.key("shift-alt-plus")
+
+    def split_window_horizontally():
+        """Splits window horizontally"""
+        actions.key("shift-alt-minus")
+
+    def split_flip():
+        """Flips the orietation of the active split"""
+        # TODO: decide whether this notification is good style
+        actions.app.notify("\"Split flip\" is not possible in windows terminal in default configuration.")
+
+    def split_window():
+        """Splits the window"""
+        # in this implementation an alias for split vertically
+        actions.key("shift-alt-plus")
+
+    def split_clear():
+        """Clears the current split"""
+        # also closes tab, because shortcut is the same
+        # and closing a split does mean something differnent that in a code editor like vs code
+        actions.key("ctrl-shift-w")
+
+    def split_clear_all():
+        """Clears all splits"""
+        actions.app.notify("TODO: decide whether to implement it at all since it either doesn't makes sense or closes the window/whole tab")
```

Comment:
This probably doesn't need to be here, just don't implement the action and leave a TODO in the file.

Reply:
ok, should I only remove the notification for the "TODO" part, or also for the other commands like "split_last" (which seem to me not possible at all)?

Reply:
If it's only about the comment with TODO and the other ones can remain, it should be done with [this commit](https://github.com/knausj85/knausj_talon/pull/711/commits/1a4d02761a11ff55a5d17a032e42107b6c4993fd)